sunkit_image/utils/tests/test_utils.py

```diff
@@ -108,10 +114,106 @@ def test_find_pixel_radii(smap):
     assert_quantity_allclose((np.max(pixel_radii)).value, known_maximum_pixel_radius)
 
     # Test that the new scale is used
-    pixel_radii = find_pixel_radii(smap, scale=2 * smap.rsun_obs)
+    pixel_radii = utils.find_pixel_radii(smap, scale=2 * smap.rsun_obs)
     assert_quantity_allclose(np.max(pixel_radii).value, known_maximum_pixel_radius / 2)
 
 
-def test_get_radial_intensity_summary():
-    # TODO: Write some tests.
-    pass
+@pytest.mark.remote_data
+def test_get_radial_intensity_summary(smap):
+
+    radial_bin_edges = u.Quantity(utils.equally_spaced_bins()) * u.R_sun
+    summary = np.mean
+
+    map_r = utils.find_pixel_radii(smap, scale=smap.rsun_obs).to(u.R_sun)
+
+    nbins = radial_bin_edges.shape[1]
+
+    lower_edge = [map_r > radial_bin_edges[0, i].to(u.R_sun) for i in range(0, nbins)]
+    upper_edge = [map_r < radial_bin_edges[1, i].to(u.R_sun) for i in range(0, nbins)]
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=RuntimeWarning)
```

Comment:
What warning happens?

Reply:
This should probably be caught with pytest.warns and matched to a message.

Reply:
Warnings arise when we take the mean of an empty slice, `summary` defaults to `np.mean` and I've left it as it is.
https://github.com/sunpy/sunkit-image/blob/a55393845f83e1a75128e567744b5a3ee3294c73/sunkit_image/utils/utils.py#L171

I've used this as a reference.

Reply:
Fair enough.

Reply:
Should I use PyTest instead?

Reply:
No, its fine.